src/components/DonationStatus/DonationStatus.js

```diff
@@ -22,43 +34,65 @@ function DonationStatus() {
 
-                    <p>Amount: NGN {donation.amount}</p>
-                    <p><a href="https://flutterwave.com/us/pricing">Payment Processor Fee</a>: NGN {donation.fee}</p>
+                    <p><strong>You donated:</strong> NGN {donation.amount}</p>
+                    <p><strong>Payment Processor Fee: </strong>NGN {donation.fee}
+                    <Button type="secondary"
+                    onClick={togglePopup2}>
+                        Details
+                    </Button></p>
 
                     {
                     donation.cbnstampdutycharge ?
-                    <p><a href='https://flutterwave.com/us/blog/product-updates/cbns-stamp-duty-charge-a-flutterwave-merchants-guide'>CBN Stamp Duty Charge</a>: NGN {donation.cbnstampdutycharge}</p> : null
+                    <p><strong>CBN Stamp Duty Charge:</strong> NGN {donation.cbnstampdutycharge}
+                    <Button type="secondary" onClick={togglePopup}>
+                        Details
+                        </Button>
+                        </p> : null
                     }
-
+                    <br/>
+                    <hr/>
                     <div>
-                        <h3>Recipients</h3>
-                        {donation.beneficiaries.map(recipient => <p>{recipient.firstName} {recipient.lastName} - NGN {recipient.amtRecvd}</p>)}
+                        <h2>You've just helped:</h2>
+                        {donation.beneficiaries.map(recipient => <p className="helped"><strong>{recipient.firstName} {recipient.lastName}</strong> - NGN {recipient.amtRecvd} ~~~~~<a href={`sms:${recipient.phone}?&body=Hey ${recipient.firstName}! My name is ${donation.donorName? donation.donorName: 'Anon'}, and I just made a donation to you via Angels Among Us. Stay healthy and keep your spirits up as we work to beat COVID-19!`}>Send {recipient.firstName} an SMS?</a></p>)}
+                        
                     </div>
-
+                    <br/>
                     <Button type="primary" link="/">Go Home</Button>
-                    <p>Share with your network!</p>
+                    <p>Share your donation story!</p>
                     <SocialShare
-                        text={"I just donated to some people who have been impacted by COVID-19. You can use this to get a randomly-generated list of three people who need your help."}
+                        text={`I've just donated ${donation.amount} to people who've been economically affected by COVID-19. You can, too, by visiting https://angelsamong.us. Anyone can be an angel!`}
                         url={window.location.origin}
                         tag={"#AngelsAmongUs"}
                     />
+                    {showPopup ? <Popup headline='About the N50 CBN Stamp Duty'
+                    summary='On September 17th, 2019, the Central Bank of Nigeria released a circular mandating a stamp duty fee of N50 on every transfer done by Nigerians worth N1000 or more.'
```

Comment:
```suggestion
                    summary='On September 17th, 2019, the Central Bank of Nigeria released a circular mandating a stamp duty fee of N50 on every Nigerian merchant transaction worth N1000 or more.'
```

Do we want to update the wording here to reflect that the fee is not "nationality" based?